sushi-config.yaml

```diff
@@ -10,7 +10,7 @@ status: active # draft | active | retired | unknown
 version: 3.2.0
 fhirVersion: 4.0.1
 copyrightYear: 2021+
-releaseLabel: ci-build # ci-build | draft | qa-preview | ballot | trial-use | release | update | normative+trial-use
+releaseLabel: release # ci-build | draft | qa-preview | ballot | trial-use | release | update | normative+trial-use
```

Comment:
You don't need to change this. It is changed automatically (temporarily) by the publish pipeline.